src/reducers/systemIntakeReducer.ts

```diff
@@ -1,14 +1,104 @@
+import cmsGovernanceTeams from 'constants/enums/cmsGovernanceTeams';
 import { STORE_SYSTEM_INTAKE } from '../constants/actions';
-import { SystemIntakeState } from '../types/systemIntake';
+import {
+  SystemIntakeState,
+  SystemIntakeForm,
+  GovernanceCollaborationTeam
+} from '../types/systemIntake';
 
 const initialState: SystemIntakeState = {};
 
 function systemIntakeReducer(state = initialState, action: any): any {
+  let systemIntake: SystemIntakeForm = {
```

Comment:
Ideally all of this data should be moved to `data/systemIntake.ts`. The reducer should be handling all of these operations.

Reply:
On that same note, since we're putting the `initialData` in `data/systemIntake.ts`, we can also delete the `initialData` from `views/SystemIntake/index.tsx` and import it from `data/systemIntake.ts`.